### PR TITLE
Foxy to galactic

### PR DESCRIPTION
**Pull request by ahcorde**
Forward port Foxy to Galactic. 

I created a Foxy release, after this PR I will make a Galactic release

### CODE REVIEW

**Review by destogl — APPROVED** [review]
Looking good! Thanks!

**Comment by ahcorde** [comment]
I will wait for the pre-commit fix

File: .github/workflows/ci-format.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (destogl) [inline .github/workflows/ci-format.yaml:1]:
```suggestion
```

**Comment by destogl** [comment]
@ahcorde I think you can merge this now.

**Pull request merged**